src/styles/blocks/section1.scss

```diff
@@ -0,0 +1,37 @@
+.section_1 {
+  margin: 0;
+  padding: 32px 16px;
+  height: 134px;
+  width: 166px;
+  text-align: center;
+}
+
+.section_1_img {
+  height: 134px;
+  width: 160px;
+  margin-bottom: 36px;
+}
```

Comment:
The class name '.section_1_img' should follow the BEM naming convention. If 'img' is an element of the 'section_1' block, it should be named '.section-1__img'.